.github/workflows/nodejs.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           WIN_CSC_LINK: ${{ secrets.WIN_CSC_LINK }}
           WIN_CSC_KEY_PASSWORD: ${{ secrets.WIN_CSC_KEY_PASSWORD }}
       - name: Update website release
-        if: contains(github.ref, '/tags/')
+        if: contains(github.ref, '/master/') && contains(github.ref, '/tags/')
```

Comment:
This won't work unfortunately. When this is triggered by a tag, the ref will be the tag, not the branch (see https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context). I'm not entirely sure how to solve this.

Reply:
Hmm, we could then check if we're on the master branch inside the script, maybe? I'll look at this next week.